driver/SDL/sdl_common.c

```diff
@@ -32,6 +32,14 @@ static lv_indev_state_t wheel_state = LV_INDEV_STATE_RELEASED;
 
 static char buf[KEYBOARD_BUFFER_SIZE];
```

Comment:
Why did you leave `buf`?  
Is it still used somewhere instead of the new `key_buffer`?

Reply:
I have implemented the new FIFO structure for SDL_TEXTINPUT also